index.html

```diff
@@ -19,8 +19,78 @@
     <div class="container">
       <div class="row">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+            <div class="jumbotron text-center">
+            <h1 class="display-4">Julian Alexander Piñeyro</h1>
+            
+            <img src="https://goo.gl/UQp1kg" class="img-fluid rounded-circle" alt="photo" width="33%" height="100%">
```

Comment:
Personally, I'd leave width and height modifiers for CSS. Not too important, but worth considering!